gix-transport/Cargo.toml

```diff
@@ -22,20 +22,42 @@ default = []
 ## If set, blocking implementations of the typical git transports become available in `crate::client`
 blocking-client = ["gix-packetline/blocking-io"]
 ## Implies `blocking-client`, and adds support for the http and https transports.
-http-client = ["base64", "gix-features/io-pipe", "blocking-client", "gix-credentials"]
+http-client = [
+    "base64",
+    "gix-features/io-pipe",
+    "blocking-client",
+    "gix-credentials",
+]
```

Comment:
I'm using zed which does auto formatting of toml 😂

I can remove them if you don't like it, but I will be away for computer for the rest of the day

Reply:
I really don't mind and believe that there is no use to 'fight' a formatter - over time, they will win :D.
The reason it's not autoformatted yet is probably because IntelliJ doesn't format TOML files for me.

I just hope Rust can align on one formatter at some point, maybe it did already, because 'formatter wars' is definitely nothing I want to be involved in ;).